### ISSUE TITLE
Error patching meshlab-git 2019.12.06.f8778c4.r0.gf8778c44-1 

### ISSUE THREAD

**Issue by ThGravo** [bartoszek/AUR-meshlab-git#1]
```
  -> Creating working copy of meshlab git repo...
Cloning into 'meshlab'...
done.
  -> Creating working copy of vcglib git repo...
Cloning into 'vcglib'...
done.
Switched to a new branch 'makepkg'
==> Starting prepare()...
==> revert upstram patch to external lib
patching file src/common/common.pro
Hunk #1 succeeded at 9 with fuzz 2 (offset 8 lines).
Hunk #2 succeeded at 37 (offset -7 lines).
Hunk #3 succeeded at 56 with fuzz 2 (offset -13 lines).
Hunk #4 FAILED at 102.
1 out of 4 hunks FAILED -- saving rejects to file src/common/common.pro.rej
patching file src/external/external.pro
patching file src/general.pri
patching file src/meshlab/meshlab.pro
Hunk #2 succeeded at 63 (offset -2 lines).
Hunk #3 succeeded at 138 (offset -19 lines).
patching file src/meshlabplugins/edit_align/edit_align.pro
Hunk #1 succeeded at 32 with fuzz 2.
patching file src/meshlabplugins/filter_plymc/plymc.pro
patching file src/meshlabplugins/io_3ds/io_3ds.pro
Hunk #2 succeeded at 25 (offset -3 lines).
patching file src/meshlabplugins/io_ctm/io_ctm.pro
Hunk #1 FAILED at 3.
1 out of 1 hunk FAILED -- saving rejects to file src/meshlabplugins/io_ctm/io_ctm.pro.rej
patching file src/meshlabplugins/render_radiance_scaling/render_radiance_scaling.pro
patching file src/meshlabserver/meshlabserver.pro
Hunk #2 succeeded at 34 with fuzz 2 (offset -10 lines).
patching file src/plugins_experimental/edit_ocme/src/ocme/ocme_builder.pro
patching file src/plugins_experimental/filter_multiscale_align/filter_multiscale_align.pro
patching file src/plugins_unsupported/edit_arc3D/edit_arc3D.pro
patching file src/plugins_unsupported/edit_fit/edit_fit.pro
patching file src/plugins_unsupported/editpaint/editpaint.pro
patching file src/plugins_unsupported/filter_rangemap/filter_rangemap.pro
patching file src/plugins_unsupported/io_epoch/io_epoch.pro
patching file src/plugins_unsupported/render_rm/render_rm.pro
patching file src/shared.pri
Hunk #2 succeeded at 23 with fuzz 1 (offset -4 lines).
==> ERROR: A failure occurred in prepare().
    Aborting...
Error making: meshlab-git
```

**Comment by bartoszek** [comment]
Yep, there's a refactoring of the build system ongoing upstream. I've just fix this yesterday, thinking I'm going to give them a day before validating my patches again.

**Comment by bartoszek** [comment]
Resolved after switch to `cmake` cca9a84bc79ed9fc8ba37a14117680de8ffb1070
